.github/workflows/ci.yml

```diff
@@ -38,3 +38,11 @@ jobs:
       env:
         TOXENV: ${{ matrix.toxenv }}
       run: tox
+    - name: Run check-keywords
+      run: |
+        pip install -r requirements/base.txt
+        cookiecutter --no-input cookiecutter-django-ida
+        cd repo_name
+        make upgrade
+        pip install -r requirements/production.txt
+        make check_keywords
```

Comment:
Just to confirm, does make check_keywords already return a non-zero exit code if it finds keywords / will this block PR merges if it finds a keyword that shouldn't be there?

Reply:
Yes, this will fail the check and action and will block the merge.